quartz/src/main/java/org/quartz/plugins/xml/XMLSchedulingDataProcessorPlugin.java

```diff
@@ -214,32 +213,29 @@ public void start(UserTransaction userTransaction) {
                 if (scanInterval > 0) {
                     getScheduler().getContext().put(JOB_INITIALIZATION_PLUGIN_NAME + '_' + getName(), this);
                 }
-                
-                Iterator<JobFile> iterator = jobFiles.values().iterator();
-                while (iterator.hasNext()) {
-                    JobFile jobFile = iterator.next();
-                
+
+                for (JobFile jobFile : jobFiles.values()) {
                     if (scanInterval > 0) {
                         String jobTriggerName = buildJobTriggerName(jobFile.getFileBasename());
                         TriggerKey tKey = new TriggerKey(jobTriggerName, JOB_INITIALIZATION_PLUGIN_NAME);
-                        
+
                         // remove preexisting job/trigger, if any
                         getScheduler().unscheduleJob(tKey);
 
                         JobDetail job = newJob().withIdentity(jobTriggerName, JOB_INITIALIZATION_PLUGIN_NAME).ofType(FileScanJob.class)
-                            .usingJobData(FileScanJob.FILE_NAME, jobFile.getFileName())
-                            .usingJobData(FileScanJob.FILE_SCAN_LISTENER_NAME, JOB_INITIALIZATION_PLUGIN_NAME + '_' + getName())
-                            .build();
+                                .usingJobData(FileScanJob.FILE_NAME, jobFile.getFileName())
```

Comment:
Not a huge fan of this kind of reformatting in a such a large PR.

Reply:
Yes  could be right, It is just a matter of preference. Do you want me to revert it?,

Reply:
I'll let @jhouserizer make the call.